media-sound/mp3gain/mp3gain-1.5.2.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Lossless mp3 normalizer with statistical analysis"
+DESCRIPTION="MP3Gain analyzes and adjusts mp3 files so that they have the same \
+volume. MP3Gain does not just do peak normalization, as many normalizers do. \
+Instead, it does some statistical analysis (using the replaygain algorithm) \
+to determine how loud the file actually sounds to the human ear.
+MP3Gain can adjust the volume in a completely lossless way by modifying the \
+mp3 file directly, without decoding and re-encoding. This works with all \
+mp3 players, i.e. no support for a special tag is required."
+HOMEPAGE="http://mp3gain.sourceforge.net"
+COPYRIGHT="2001-2009 David Robinson and Glen Sawyer
+	2002 John Zitterkopf"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/mp3gain/mp3gain-${portVersion//./_}-src.zip"
```

Comment:
http -> https
(SF also accepts to serve the downloads through https since 1 or 2 years.)